--- dataloader.py.orig	2026-01-30 19:29:45
+++ dataloader.py	2026-01-30 19:30:20
@@ -87,14 +88,14 @@
 
     # extract mel-spectrogram
     if hparams.feats == 'mel':
-        spect = librosa.feature.melspectrogram(wav,
+        spect = librosa.feature.melspectrogram(y=wav,
                                                sr=sr,
                                                n_fft=hparams.n_fft,
                                                hop_length=hparams.hop_length,
                                                n_mels=hparams.rnn_input_size)
     # extract mfcc
     elif hparams.feats == 'mfcc':
-        spect = librosa.feature.mfcc(wav,
+        spect = librosa.feature.mfcc(y=wav,
                                      sr=sr,
                                      n_fft=hparams.n_fft,
                                      hop_length=hparams.hop_length,
@@ -208,7 +209,10 @@
         raise NotImplementedError
 
     def process_file(self, wav_path):
-        phn_path = wav_path.replace("wav", "phn")
+        phn_path = wav_path.replace("WAV", "PHN")
 
         # load audio
         spect = extract_features(wav_path, self.hparams)
@@ -235,7 +239,7 @@
 
     def _make_dataset(self):
         files = []
-        wavs = list(iter_find_files(self.wav_path, "*.wav"))
+        wavs = list(iter_find_files(self.wav_path, "*.WAV"))
         if self.hparams.devrun:
             wavs = wavs[:self.hparams.devrun_size]
 
@@ -265,10 +269,19 @@
         self.data = self._make_dataset()
 
     @staticmethod
-    def get_datasets(hparams):
-        train_dataset = TimitDataset(join(hparams.wav_path, 'train'),
-        test_dataset  = TimitDataset(join(hparams.wav_path, 'test'),
+        train_dataset = TimitDataset(join(hparams.wav_path, 'TRAIN'),
+        test_dataset  = TimitDataset(join(hparams.wav_path, 'TEST'),
 
         train_len   = len(train_dataset)
